src/internal/github/git_test.go

```diff
@@ -0,0 +1,55 @@
+package github
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseTagsFromStdout(t *testing.T) {
+	cases := map[string]struct {
+		input       []string
+		expected    []string
+		expectedErr error
+	}{
+		"Simple Tag": {
```

Comment:
It would be cool if we could add tests for empty string inputs (for fresh repos)